client/src/pages/Login/Login.tsx

```diff
@@ -5,6 +5,7 @@ import login from '../../helpers/APICalls/login';
 import LoginForm from './LoginForm/LoginForm';
 import { useAuth } from '../../context/useAuthContext';
 import { useSnackBar } from '../../context/useSnackbarContext';
+import Paper from '@material-ui/core';
```

Comment:
This seems like an irrelevant change, can we exclude it?

Reply:
Done